packages/generator/src/edmx-parser/v2/edmx-parser.ts

```diff
@@ -35,8 +35,17 @@ export function parseAssociationSets(root: any): EdmxAssociationSet[] {
 }
 
 export function parseFunctionImports(root: any): EdmxFunctionImport[] {
-  return getPropertyFromEntityContainer(root, 'FunctionImport').map(f => ({
+  const functionImports: EdmxFunctionImport[] = getPropertyFromEntityContainer(
+    root,
+    'FunctionImport'
+  ).map(f => ({
     ...f,
     Parameter: forceArray(f.Parameter)
   }));
+  return functionImports.map(f => {
+    f.Parameter.forEach(p => {
+      p.Nullable = !p.Nullable ? 'true' : p.Nullable;
```

Comment:
I think we already have a function for this. Could you please check if this is applicable here?

Reply:
We have a isNullableParameter which does not default to true.In the generator-utls there are also other method but non seem to fist the: 
- give value
- if property not present return true.